### PR TITLE
provide a definition for the close connection option under 1.1 tear-down

### LINKED ISSUE #678: close connection item lost its definition
The close connection item used to be defined in Connection as

```
   The "close" connection option is defined for a sender to signal that
   this connection will be closed after completion of the response.  For
   example,

     Connection: close

   in either the request or the response header fields indicates that
   the sender is going to close the connection after the current
   request/response is complete (Section 6.6).
```

whereas now its effect is described in Messaging without definition.

I think it should be defined somewhere.

### PR DESCRIPTION
**Pull request by royfielding**
Fixes #678

### CODE REVIEW

File: draft-ietf-httpbis-messaging-latest.xml [hunk 1]
```diff
@@ -1785,39 +1777,58 @@ http://www.example.org:8080
...
 <t>
-   A server that receives a "<x:ref>close</x:ref>" connection option &MUST;
-   initiate a close of the connection (see below) after it sends the
-   final response to the request that contained "close".
-   The server &SHOULD; send a "close" connection option
-   in its final response on that connection. The server &MUST-NOT; process
-   any further requests received on that connection.
+   A server that receives a close connection option &MUST;
+   initiate closure of the connection (see below) after it sends the
+   final response to the request that contained the close.
```
Comment (mnot) [inline draft-ietf-httpbis-messaging-latest.xml:1815]:
```suggestion
   final response to the request that contained the connection option.
```

Comment (mnot) [reply draft-ietf-httpbis-messaging-latest.xml:1815]: 'the close' is a bit weird.

File: draft-ietf-httpbis-messaging-latest.xml [hunk 2]
```diff
@@ -1785,39 +1777,58 @@ http://www.example.org:8080
...
+   on that connection. The server &MUST-NOT; process any further requests
+   received on that connection.
 </t>
 <t>
-   A server that sends a "<x:ref>close</x:ref>" connection option &MUST;
-   initiate a close of the connection (see below) after it sends the
-   response containing "close". The server &MUST-NOT; process
+   A server that sends a close connection option &MUST;
+   initiate closure of the connection (see below) after it sends the
+   response containing the close. The server &MUST-NOT; process
```
Comment (mnot) [inline draft-ietf-httpbis-messaging-latest.xml:1823]:
```suggestion
   response containing the connection option. The server &MUST-NOT; process
```

**Review by mnot — APPROVED** [review]
LGTM, except for 'the close' being a bit weird (but I don't feel that strongly about it; suggestions for replacement inline).

File: draft-ietf-httpbis-messaging-latest.xml (hunk 1 above)
Comment (reschke) [inline draft-ietf-httpbis-messaging-latest.xml:1815]:
```suggestion
   final response to the request that contained the close connection option.
```

File: draft-ietf-httpbis-messaging-latest.xml (hunk 2 above)
Comment (reschke) [inline draft-ietf-httpbis-messaging-latest.xml:1823]:
```suggestion
   response containing the close connection option. The server &MUST-NOT; process
```

File: draft-ietf-httpbis-messaging-latest.xml [hunk 3]
```diff
@@ -1785,39 +1777,58 @@ http://www.example.org:8080
...
    any further requests received on that connection.
 </t>
 <t>
-   A client that receives a "<x:ref>close</x:ref>" connection option &MUST;
+   A client that receives a close connection option &MUST;
    cease sending requests on that connection and close the connection
-   after reading the response message containing the "close"; if additional
-   pipelined requests had been sent on the connection, the client &SHOULD-NOT;
-   assume that they will be processed by the server.
+   after reading the response message containing the close; if additional
```
Comment (reschke) [inline draft-ietf-httpbis-messaging-latest.xml:1829]:
```suggestion
   after reading the response message containing the close connection option; if additional
```

**Review by reschke — COMMENTED** [review]
looks good except as noted by @mnot, the bare _close_ strings look weird.

File: draft-ietf-httpbis-messaging-latest.xml [hunk 4]
```diff
@@ -1785,39 +1777,58 @@ TE: trailers, deflate;q=0.5
...
 </t>
 <t>
-   A client that receives a "<x:ref>close</x:ref>" connection option &MUST;
+   A client that receives a close connection option &MUST;
    cease sending requests on that connection and close the connection
-   after reading the response message containing the "close"; if additional
-   pipelined requests had been sent on the connection, the client &SHOULD-NOT;
-   assume that they will be processed by the server.
+   after reading the response message containing the close connection option; if additional
+   pipelined requests had been sent on the connection,
```
Comment (royfielding) [inline draft-ietf-httpbis-messaging-latest.xml:1830]:
```suggestion
   after reading the response message containing the close connection option;
   if additional pipelined requests had been sent on the connection,
```

**Pull request merged**